api/src/models/mod.rs

```diff
@@ -27,7 +27,7 @@
 pub use app_name::{AppName, AppNameError};
 pub use image::Image;
 pub use logs_chunks::LogChunk;
-pub use service::{ServiceBuilder, ServiceBuilderError};
+pub use service::{ContainerType, ServiceBuilder, ServiceBuilderError};
```

Comment:
Just because I can't seem to find it: Is the additional import used anywhere?

Reply:
It will be used in `crate::apps::DeploymentUnit` and I started to reexport the structs a while ago and `ContainerType` was missing. With this change you can just write `crate::models::ContainerType`.